tests/testthat/test-gee-tidiers.R

```diff
@@ -0,0 +1,28 @@
+context("gee-gee")
+
+skip_if_not_installed("modeltests")
+library(modeltests)
+
+test_that("gee tidier arguments", {
+  check_arguments(tidy.gee)
+  check_arguments(glance.gee)
+})
+
+g1 <- gee::gee(mpg ~ wt + hp, id = cyl, data = mtcars, corstr = "exchangeable")
+
+# Tidy
```

Comment:
I think this comment (and similarly for `glance`) is probably unnecessary as it is already very clear what your tests do :)